owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/status/GetRemoteStatusOperation.kt

```diff
@@ -46,115 +41,36 @@ import javax.net.ssl.SSLException
  * @author Abel García de Prada
  */
 class GetRemoteStatusOperation : RemoteOperation<OwnCloudVersion>() {
-    private lateinit var latestResult: RemoteOperationResult<OwnCloudVersion>
+    companion object {
+        const val HTTPS_SCHEME = "https"
```

Comment:
I guess you'll get this form other places.
Please double check.

Reply:
I'm not sue if OC has it always on top of a class, maybe it's on bottom.
Please double check

Reply:
done